Jenkinsfile

```diff
@@ -31,13 +31,13 @@ node('linux && docker') {
         junit 'packages/*/reports/*.xml'
       }
 
-      //stage('Cypress Test') {
-      //  sh 'apt-get -y install libgtk2.0-0 libgtk-3-0 libnotify-dev libgconf-2-4 libgbm-dev libnss3 libasound2 xauth xvfb'
-      //  sh 'rm -rf /var/lib/apt/lists/*'
-      //  sh 'cd packages/atomic && npx cypress install'
-      //  sh 'npm start & npx wait-on http://localhost:3333'
-      //  sh 'NO_COLOR=1 npm run cypress:test'
-      //}
+      stage('Cypress Test') {
+       sh 'apt-get -y install libgtk2.0-0 libgtk-3-0 libnotify-dev libgconf-2-4 libgbm-dev libnss3 libasound2 xauth xvfb'
+       sh 'rm -rf /var/lib/apt/lists/*'
+       sh 'cd packages/atomic && npx cypress install'
+       sh 'cd packages/atomic && npm start & npx wait-on http://localhost:3333'
```

Comment:
no, it doesn't stay inside the packages/atomic, it goes back to main directory somehow, as when I tried on jenkin, it started quantic as well, and our docker image right now doens't have sfdx, so it will break the build